nature/migrations/0005_rename_event_models.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.13 on 2018-06-11 11:40
+from __future__ import unicode_literals
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
```

Comment:
This kind of migrations are irreversible, IMHO it is safer to modify the initial migration. This way we can make sure that all the non-schema changes are in the initial setup and faked.

I think we going to do more renamings in the future, some of the terms may have better translations in the domain context. It would be good that we put all these translation corrections into the initial migration.

Reply:
Alright! As I asked above: is it correct to assume that there will not be a problem if this is run on the current production code? Will everything be reimported again there? I'm just wondering how the changes will be made to the production environment without having new migrations...

Reply:
In this case the table names and column names (for fields) are defined already in the model so there would be no changes to the actual database in this case.  But of course, this can be added to the initial migration if the migrations are run from scratch again.

Reply:
>  But of course, this can be added to the initial migration if the migrations are run from scratch again.

This is very likely to happen, the database in production is also an old one and there will be a new database dump containing the latest data. And all the migrations (and sql scripts) need be run again from scratch.

Reply:
Alright, then I'll move everything over to the initial migrations 👍 😄